src/pages/home/index.js

```diff
@@ -6,9 +6,9 @@ import { cms } from '../../client'
 import Home from './styled'
 import mural from './mural.png'
 import NavBar from '../../components/navbar'
-import ModalGallery from '../../components/galery-modal'
-import fotoNossaCasa from './nossacasa.png'
-import Parceires from '../../components/parceires'
+import ModalGallery from '../../components/modal-gallery'
+import photoOurHouse from './nossacasa.png'
```

Comment:
Como regra geral, não somente em código, mas em tradução português para inglês e vice versa, não se traduz nome próprio:

Talvez um nome mais apropriado seja:

```suggestion
import nossaCasaImage from './nossacasa.png'
```